.env_sample

```diff
@@ -0,0 +1,19 @@
+# APP_HOST is the host of your app, if you deploy the app change to your domain.
+export APP_HOST=localhost
+
+# GITHUB_APP_ID is found in settings of your GitHub app under General > About
+export GITHUB_APP_ID=YOUR_GITHUB_APP_ID
+
+# GITHUB_APP_NAME what you called your app.
+export GITHUB_APP_NAME=NAME_OF_THE_GITHUB_APP
+
+# SECRET_KEY_BASE is required for Auth Cookie:
+export SECRET_KEY_BASE=run:mix_phx.gen.secret-in-terminal-and-paste-result-here
+
+# AWS:
+export AWS_ACCESS_KEY_ID=aws-access-key
```

Comment:
Tiny observation but perhaps `aws-access-key` could say `aws-access-key-id` in line with the key name?

Reply:
thanks for your attention to detail @Cleop 🎉